server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexVersionModule.scala

```diff
@@ -53,10 +52,10 @@ class AddressIndexVersionModule @Inject()
       }
 
     // yes, it is blocking, but it only does this request once and there is also timeout in case it goes wrong
-    val indexes: Either[RequestFailure, RequestSuccess[IndexAliases]] = Await.result(requestForIndexes, 10 seconds)
+    val indexes = Try(Await.result(requestForIndexes, 10 seconds)).toEither
```

Comment:
Shouldn't we make this timeout an env variable?

Reply:
I'm hoping with the new versioning arrangements we can ditch this call completely (this will mean the default service account will not need "manage" access when we turn on the security).